test_scripts/Defects/4_5/1880_DefaultAndRPCTimeoutGENERIC_ERROR.lua

```diff
@@ -313,17 +351,17 @@ local function PerformInteraction(params, self)
   local RespTimeout
   if params.timeout then
     PIDuration = params.timeout
-    RespTimeout = DefaultTimeout + 2*PIDuration
+    RespTimeout = GeneralTimeout + 2*PIDuration
```

Comment:
```suggestion
    RespTimeout = GeneralTimeout + PIDuration
```
I believe that the double timeout is redundant

Reply:
Please find update in [a92da80](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/174/commits/a92da8016cffbf484eb493195cf5404c3dd70ce2)